### ISSUE TITLE
Example of use with mzML

### ISSUE THREAD

**Issue by lazear** [bibs-lab/SpecGlobX#2]
Hi, 

Is it possible for you to upload an example mzML and CSV input? I cannot seem to get the program to work, despite it saying "Done!"

My csv looks like: 
```
peptide;scannr
IGASFLQRFFKFSR;3423
GLVGIGLFALAHAAFSAAQHR;5656
NIPLLFLQNITGFMVGR;5829
TGPAATTLPDGAAAESLVESSEVAVIGFFK;6534
EVATAIRGAIILAKLSIVPVR;4159
```

And I am setting the peptide column to 1 and spectrum ID to 2. 

```
Peaks filter choosen : Rel. intensity threshold (%)(0) --> 1
Multiprocesing : true --> 10 thread to use
config.properties file not found in the JAR foldermzML File Loaded
Indexation of spectra
Indexation DONE !
=====STARTING ALIGNMENTS=====
There are 1640 alignments to do !
10 threads used to execute alignments !
Done!
```

But the output.csv file contains no rows of data

**Comment by vlollier** [comment]
Hi @lazear , actually the current version of SpecGlobX expects PSM csv input file in a reversed order as yours: Title column first, followed by peptide sequence. 
You can browse the test folder of this github repository for an example of data.

**Comment by lazear** [comment]
I will give it a shot. The GUI and docs implied that column ordering doesn't matter, since you have to manually set the column numbers anyway (which I was doing)

**Comment by vlollier** [comment]
Ok. I thought you were using default parameters on command line. My blame!
However, process logs shown "config.properties file not found ". If the progress bar in GUI also stays at the middle status, try to launch the program from the folder containing this file (at root repos)  or copy & paste this file into target folder (with the jarfile).

**Comment by dominique-tessier** [comment]
Hi Michael,
Thank you very much for your comments. I ran a series of tests following your configuration, trying to catch SpecGlobX at fault. Unfortunately, I succeeded! You've probably identified a bug in SpecGlobX related to thread management between jmzReader and SpecGlobX. The issue seems to happen with a small number of PSMs to align and/or a large number of threads. SpecGlobX terminates before the results can be written.
Another problem can lead to a missing result file: if the result file is already open elsewhere (in EXCEL, for example), in the current release, no appropriate message is written in the log window to warn the user.
I'll fix these problems as soon as possible in a new release.
